plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraType.java

```diff
@@ -165,7 +172,7 @@ public String getName()
         return kind.name();
     }
 
-    public static Optional<CassandraType> toCassandraType(DataType dataType)
+    public static Optional<CassandraType> toCassandraType(DataType dataType, TypeManager typeManager)
```

Comment:
Can you please consider extracting the static methods from `CassandraType` to a Guice bean which has the `TypeManager typeManager` as `private final` field ?

This would leave `CassandraType` a simple POJO and the newly created bean could get eventually further dependencies, but further expose a simple public interface to the client classes.

Reply:
@chen-ni You can refer to https://github.com/ebyhr/trino/commit/10742d16940f9b970e0b05bce37d766d35f5d8f4 as the example. Please note that extracting those methods in a commit may make git diff dirty. You may need additional commits for keeping clean diff.

Reply:
@findinpath @ebyhr Thanks, I'll give it a try  :)

Reply:
@ebyhr I just removed my old commit, cherry-picked your refactoring commit, then put the `INET -> IPADDRESS` type mapping part based on it. So now we have 2 commits, 1 for the refactoring and 1 for the type mapping. Hopefully it will work.

Reply:
@ebyhr @findinpath Integration with @ebyhr 's refactoring looks fine. Please review when possible :)